sample/src/main/java/app/cash/paparazzi/sample/ResourcesDemoView.kt

```diff
@@ -0,0 +1,89 @@
+package app.cash.paparazzi.sample
+
+import android.content.Context
+import android.graphics.Color
+import android.icu.text.MessageFormat
+import android.text.Html
+import android.util.TypedValue
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.annotation.ColorInt
+import androidx.annotation.DrawableRes
+
+class ResourcesDemoView(context: Context) : LinearLayout(context) {
+  init {
+    setBackgroundColor(Color.WHITE)
+    orientation = LinearLayout.VERTICAL
+    layoutParams = ViewGroup.LayoutParams(
+      ViewGroup.LayoutParams.MATCH_PARENT,
+      ViewGroup.LayoutParams.MATCH_PARENT
+    )
+    addImageView(R.drawable.camera)
+    addImageView(R.drawable.ic_android_black_24dp)
+    addTextView(resources.getBoolean(R.bool.adjust_view_bounds).toString())
+    addTextView(
+      text = "Color",
+      backgroundColor = resources.getColor(R.color.keypadGreen, null)
+    )
+    addTextView(resources.getString(R.string.string_escaped_chars))
+    addTextView("Height: ${resources.getDimension(R.dimen.textview_height)}")
+    addTextView("Max speed: ${context.resources.getInteger(R.integer.max_speed)}")
+    addTextView(context.resources.getQuantityString(R.plurals.plural_name, 0))
+    addTextView(context.resources.getQuantityString(R.plurals.plural_name, 1))
+    addTextView(context.resources.getQuantityString(R.plurals.plural_name, 2))
+    addTextView(resources.getString(R.string.string_name))
+    addTextView(MessageFormat.format(context.resources.getString(R.string.string_name_xliff), 5))
+    addTextView(
+      Html.fromHtml(
+        resources.getString(R.string.string_name_html),
+        Html.FROM_HTML_MODE_LEGACY
+      )
+    )
```

Comment:
nice, Html parsing 🎉

Reply:
yea, however and unfortunately: this doesn't work yet (no longer chops off the substring past the closing tag, but doesn't honor bold/italics/etc.) despite it working in the IDE.